src/hal/src/device.rs

```diff
@@ -6,7 +6,7 @@
 //! and managing graphics resources such as buffers, images and memory.
 //!
 //! The `Adapter` and `Device` types are very similar to the Vulkan concept of
-//! "physical devices" vs. "logical devices"; an `Adapter` is single GPU
+//! "logical devices" vs. "physical devices"; an `Adapter` is single GPU
```

Comment:
I don't think this change is correct. The line above says "Adapter" followed by "Device, so here it should be "physical device" followed by "logical device". The old comment is correct, therefore.

Reply:
Here’s to say the “Adapter” equals to “physical device” and “Device” is “logical device”?

Reply:
That is correct, yes

Reply:
Thank you.

Reply:
@kvark what do you think about aligning the naming with Vulkan here?